src/eduid_idp/idp_actions.py

```diff
@@ -122,7 +121,5 @@ def add_idp_initiated_actions(context: IdPContext, user: IdPUser, ticket: SSOLog
     :param user: the authenticating user
     :param ticket: the SSO login data
     """
-    if 'mfa' in context.config.action_plugins:
-        eduid_idp.mfa_action.add_actions(context, user, ticket)
-    if 'tou' in context.config.action_plugins:
-        eduid_idp.tou_action.add_actions(context, user, ticket)
+    for action in context.config.action_plugins:
```

Comment:
Do we need to make this change? What's the gain? Every time we've done fancy things with dynamic calling of plugins-like things, we've gotten some kind of pain for it down the line.

Reply:
The idea was to have the least places to touch when a new plugin is added. But I agree with you, I'll revert this.

Reply:
done